src/network/connectionthreads.cpp

```diff
@@ -717,13 +717,15 @@ void ConnectionSendThread::sendPackets(float dtime)
 					channel.outgoing_reliables_sent.size()
 					< channel.getWindowSize() &&
 					peer->m_increment_packets_remaining > 0) {
-				BufferedPacket p = channel.queued_reliables.front();
+				BufferedPacket p = std::move(channel.queued_reliables.front());
 				channel.queued_reliables.pop();
+
 				LOG(dout_con << m_connection->getDesc()
 					<< " INFO: sending a queued reliable packet "
 					<< " channel: " << i
 					<< ", seqnum: " << readU16(&p.data[BASE_HEADER_SIZE + 1])
 					<< std::endl);
+
 				sendAsPacketReliable(p, &channel);
```

Comment:
```suggestion
				sendAsPacketReliable(std::move(p), &channel);
```
(You could also here use an rvalue ref and pop() after send to move once fewer.)